docs/compilation/common_errors.md

```diff
@@ -0,0 +1,81 @@
+# Common errors
+
+In this document, we list the most common errors, and mention how the user can fix them.
+
+## 1. Could not find a version that satisfies the requirement concrete-python (from versions: none)
+
+**Error message**: `Could not find a version that satisfies the requirement concrete-python (from versions: none)`
+
+**Cause**: The installation does not work fine for you.
+
+**Possible solutions**:
+- Be sure that you use a supported Python version (currently from 3.8 to 3.11, included)
+- Check you have done `pip install -U pip wheel setuptools` before
+- Consider adding a `--extra-index-url https://pypi.zama.ai`
+- Concrete requires glibc>=2.28, be sure to have a sufficiently recent version
+
+## 2. Only integers are supported
+
+**Error message**: `RuntimeError: Function you are trying to compile cannot be compiled` with extra information `only integers are supported`
+
+**Cause**: This error can occur if parts of your program contain graphs which are not from integer to integer
+
+**Possible solutions**:
+- It is possible to use floats as intermediate values (see the [documentation](../core-features/floating_points.md#floating-points-as-intermediate-values)) but always, inputs and outputs must be integers. So, consider adding ways to convert to integers, such as `.astype(np.uint64)`
+
+## 3. No parameters found
+
+**Error message**: `NoParametersFound`
+
+**Cause**: The optimizer was not able to find cryptographic parameters for the circuit, which are both secure and correct
+
+**Possible solutions**:
+- Try to simplify your circuit
+- Use smaller weights,
+- Add intermediate PBS to reduce the noise, with identity function `fhe.univariate(lambda x: x)`
```

Comment:
We have `fhe.identity(x)` which is better in most cases (though it uses reinterpret so it wouldn't work on CRT)

Reply:
Maybe we could keep it like this: it's for non power users so let's keep something which is less optimized but works more often, ok?